src/default-markup.ts

```diff
@@ -0,0 +1,235 @@
+      `,
+      // derek, you mentioned adding blur up in onClientEntry, but I'll leave this here for now:
+      imagePlaceholder: `
+        position: absolute;
+        top: 0; bottom: 0; left: 0; right: 0;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        object-position: center center;
+        filter: blur(50px);
+      `,
+      svgPlaceholder: `
+        position: absolute;
+        top: 0; bottom: 0; left: 0; right: 0;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        object-position: center center;
+      `,
+      imageTag: `
+        position: absolute;
+        top: 0; bottom: 0; left: 0; right: 0;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        object-position: center center;
+      `,
+    },
+    fixed: {
+      imageWrapper: `
+        position: relative;
+        overflow: hidden;
+        display: inline-block;
+        width: ${props.width || 'auto'};
+        height: ${props.height || 'auto'};
+      `,
+      solidPlaceholder: `
+        background-color: ${props.backgroundColor ||
+          props.dominateColor ||
+          'inherit'};
+        width: ${props.width || 'auto'};
+        height: ${props.height || 'auto'};
+      `,
+      imagePlaceholder: `
+        width: ${props.width || 'auto'};
+        height: ${props.height || 'auto'};
+        object-fit: cover;
+        object-position: center center;
+        filter: blur(50px);
+      `,
+      svgPlaceholder: `
+        position: absolute;
+        top: 0; bottom: 0; left: 0; right: 0;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        object-position: center center;
+      `,
+      imageTag: `
+        position: absolute;
+        top: 0; bottom: 0; left: 0; right: 0;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        object-position: center center;
+      `,
+    },
+  }
+
+  if (props.sharpMethod === 'fluid') {
+    return `
+      <div class="gria-image-wrapper" style="${styles.fluid.imageWrapper}">
+
+        ${comment('preserve the aspect ratio')}
+        <div class="gria-image-padding" style="${
+          styles.fluid.imagePadding
+        }"></div>
+
+        ${comment('show a solid background color.')}
+        <div
+          class="gria-solid-placeholder"
+          title="${alt}"
+          style="${styles.fluid.solidPlaceholder}">
+        </div>
+
+        ${comment('show the blurry base64 image.')}
+        <img
+          class="gria-base64-placeholder"
+          src="${props.base64}"
+          title="${alt}"
+          alt="${alt}"
+          style="${styles.fluid.imagePlaceholder}"
+        >
+
+        ${comment('show a traced SVG image.')}
+        <img
+          class="gria-tracedSVG-placeholder"
+          src="${props.tracedSVG}"
+          title="${alt}"
+          alt="${alt}"
+          style="${styles.fluid.svgPlaceholder}"
+        >
```

Comment:
Depending users' preference, we'll only need one or two of these at the same time right? Would it make sense to toggle them depending on the passed in options?